### DISCUSSION TITLE
Password not working after upgrade. Crashplan was last started in 2020.

### DISCUSSION THREAD

**Discussion by PerJoh24** [jlesage/docker-crashplan-pro#463] [Q&A]
Hi,

I've restarted my Synology/Crashplan after having it down for four years. I've upgraded the DSM and the crashplan package. The package looks to be working and connecting to Crashplan. 
However, my old password (which I'm sure of) doesn't work. I just get the old access denied.

Is there anyone out there that can help?

Thanks,
P

**Comment by jlesage** [comment]
Are you using the same password that you would use to connect to crashplan.com ?

**Reply by PerJoh24** [reply]
No I have different passwords for the site and my Synology. However, I had managed to update the passwords but hadn't saved it i my KeyPass store. When I guessed a little bit I was able to log in. 

Now I just have to re-connect my Crashplan installation so that it has the proper rights to the Synology file system. At this point my Crashplan installation doesn't show any files to back up on the Synology.

Thanks,
Per
